src/datachain/lib/dc/parquet.py

```diff
@@ -26,8 +26,14 @@ def read_parquet(
     """Generate chain from parquet files.
 
     Parameters:
-        path: Storage URI with directory. URI must start with storage prefix such
-            as `s3://`, `gs://`, `az://` or "file:///".
+        path: Storage path(s) or URI(s). Can be a local path or start with a
+            storage prefix like `s3://`, `gs://`, `az://`, `hf://` or "file:///".
```

Comment:
May be to also add `http/https` too if we want to merge this PR? https://github.com/iterative/datachain/pull/1331

```suggestion
            storage prefix like `s3://`, `gs://`, `az://`, `hf://`, `https://` or "file:///".
```

or even may be make it as list here 🤔